contracts/projectRegistry/ProjectRegistry.sol

```diff
@@ -91,13 +103,23 @@ contract ProjectRegistry is Initializable {
     /**
      * @notice Updates Metadata for singe project
      * @param projectID ID of previously created project
-     * @param metadata Updated pointer to external metadata
+     * @param metadata Updated pointer to project metadata
      */
     function updateProjectMetadata(uint256 projectID, MetaPtr calldata metadata) external onlyProjectOwner(projectID) {
         projects[projectID].metadata = metadata;
         emit MetadataUpdated(projectID, metadata);
     }
 
+    /**
+     * @notice Updates Metadata for singe project
+     * @param projectID ID of previously created project
+     * @param programMetadata Updated pointer to project programMetadata
+     */
+    function updateProgramMetadata(uint256 projectID, MetaPtr calldata programMetadata) external onlyProjectOwner(projectID) {
```

Comment:
`updateProgramMetadata` -> we could use updateProjectMetadata and have the owner use that function to update both metadata (this kinda makes sense but idk)

Reply:
It does make sense, how are you thinking of doing this?

🤔

```solidity
enum PtrType {
    Project,
    Program
}

function updateMetadata(uint256 projectID, MetaPtr calldata metadata, PtrType metadataType) external onlyProjectOwner(projectID) {
    if (metadataType === PtrType.Project) {
        projects[projectID].metadata = metadata;
        emit MetadataUpdated(projectID, metadata);
    }
    if (metadataType === PtrType.Program) {
        projects[projectID].programMetadata = programMetadata;
        emit ProgramMetadataUpdated(projectID, programMetadata);
    }
}
```

Reply:
Yup something along those lines but i think this makes it a little uglier cause we need to maintain an enum and the update Metadata is responsible for 2 things which feels icky